core/tests/unit/ldml/keyboards/meson.build

```diff
@@ -26,6 +26,7 @@ tests_without_testdata = [
   'k_010_mt',
   'k_011_mt_iso',
   'k_012_other',
+  'k_015_importlocal',
```

Comment:
There's a real argument for moving these source files out of core and only having .kmx files here:

1. removes the dependency on kmc-ldml from core test.
2. narrows the focus of the unit test to core's actual area of responsibility, e.g. local imports should not really be relevant to core -- that should be tested as part of the compiler as the .kmx doesn't reference the imports.

We can have the sources in developer and have the .kmx written to this folder as part of kmc-ldml build and/or test, so git would pick up if there are changes automatically.

Reply:
So, the same test file is in 3 places now:
- LDML reader - this is the main test, and closest to where the import is processed.
- kmc-ldml - to verify that the keys compiler seems the imports properly and that the import status propagates properly to the compiler (useful for warnings).
- core - this is an end to end test including that the ldml_processor ends up with the expected behavior due to the imports.

So to your point 2, I could simply *remove* the second and third instances of this test file and say - it's been unit tested in the XML reader, the other cases aren't needed.  Maybe that's OK in this case (as you said, entirely a compiler issue). But there are other cases where a compiler test (say, differing layers and modifiers) also needs to be tested against the engine in core.

To your point 1, I think it probably makes sense to remove that dependency. it would also help us be cognizant of any accidental binary format changes.  Having smart rebuilding of only the changed .kmx would be nice for edit/compile/test cycles (even though I'd need to run a build in two places - kmc-ldml and in core).

Reply:
For now i've removed this test from core.

Reply:
> To your point 1, I think it probably makes sense to remove that dependency. it would also help us be cognizant of any accidental binary format changes. Having smart rebuilding of only the changed .kmx would be nice for edit/compile/test cycles (even though I'd need to run a build in two places - kmc-ldml and in core).

Yes I think so. Perhaps if the unit test in the compiler updates .kmx binaries in the Core folder, then reports a failure if any git changes are found, we could pickup on the changes that way?

A 19.0 reorganization perhaps.